spec/javascripts/actions/story_spec.js

```diff
@@ -320,28 +323,35 @@ describe("Story Actions", () => {
         })(fakeDispatch, fakeGetState, { Story: FakeStory });
       });
 
-      it("dispatches storyFailure", () => {
+      it('dispatches storyFailure', () => {
         expect(fakeDispatch).toHaveBeenCalledWith(
           Story.storyFailure(updatedStory.id, error)
         );
       });
 
-      it("do not dispatch sortStoriesSuccess", () => {
-        expect(fakeDispatch).not.toHaveBeenCalledWith(
-          Story.sortStoriesSuccess(updatedStory)
-        );
-      });
+      // it('do not dispatch sortStories', () => {
+      //   // expect(fakeDispatch).not.toHaveBeenCalledWith(
+      //   //   Story.sortStories(updatedStory)
+      //   // );
+      //   expect(fakeDispatch).toHaveBeenCalledWith(
+      //     Story.sortStories(storiesWithUpdatedPositions, from)
+      //   );
+
+      //   expect(fakeDispatch).not.toHaveBeenCalledWith(
+      //     Story.sortStories(updatedStory, from)
+      //   );
+      // });
```

Comment:
Are these comments supposed to be here? Maybe we could use `.skip` instead if that's the case

Reply:
forgot to remove kkkk